website_snippet_product_category/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Active public categories based on previous is_published field
+    """
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE product_public_category
+        SET is_published_in_snippet = is_published
```

Comment:
You need to rename the column in pre instead, or when getting here, it won't exist (normal ORM drops it).